workflows/prognostic_c48_run/tests/_regtest_outputs/test_regression.test_fv3run_diagnostic_outputs_check_variables[nudging].out

```diff
@@ -4,7 +4,7 @@ area: f672e2d555d61777324a730a427f4c22
 cnvprcp_after_physics: a0279aa038e7f90e25d8828bb1fc853b
 cnvprcp_after_python: a0279aa038e7f90e25d8828bb1fc853b
 evaporation: 5daea270430161b4f40aac01c110be63
-net_heating_due_to_nudging: 9ea773b10d712efc2f135505fdc81dd7
+net_heating_due_to_nudging: 3c30e5e85a39c89e346f171c04eb1380
```

Comment:
because both the `sklearn` and `keras` mock ML models predict 0 for all the heatings, the nudging run is the only that catches this difference in net heating diagnostic.